src/smart-components/role/add-role-new/review.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-form-api';
+import {
+    Stack,
+    StackItem,
+    Grid,
+    GridItem,
+    Text,
+    TextVariants
+} from '@patternfly/react-core';
+import './review.scss';
+
+const mockData = [
```

Comment:
Is this intended? We don't have the data from API yet?

Reply:
Yep, the data presented here will be filed in previous step which is not done yet. And api is probably not ready yet also.